### ISSUE TITLE
Steam in-home streaming

### ISSUE THREAD

**Issue by ccomly** [d07RiV/ShrineTips#2]
Im running PoE with steam in-home streaming. And i can see the tooltip for a breif 0.01 seconds. I only see the border of the tool-tip before if vanishes. I think it has something to do with the streaming, cause it

kind of automatically always returns focus to the game being streamed, and does so very fast. Would be cool if it worked with in-home streaming.

edit:

I tried adding shrinetips.exe to steam, sometimes u can add applications and stream them with steam this way, like notepad and get a full screen desktop. It starts shrinetips.exe but just stops at a black screen, probably cause shrine tips doesn't have a window. I then manually started poe from teamviewer while "streaming" shrinetips.exe, and its switches to poe. However the tooltip goes away really fast like before.

Maybe some kind of "force on top" or giving it a permanent window that sits ontop. Might work idk.

**Comment by d07RiV** [comment]
Yeah I keep hearing about this problem, but I wasn't able to reproduce it. It is a topmost window already.

Can you check if it goes behind the game window or disappears completely? I believe you could check that by using windowed mode and positioning the cursor such that the tooltip will stick outside of the game window.

**Comment by ccomly** [comment]
yep i hooked up a monitor on the host, changed resolution to 800x600 windowed, and put a rare in far right side of inventory and i could see half the tooltip behind  the window
